source/development/wazuh-cluster.rst

```diff
@@ -83,21 +83,43 @@ The image below shows a schema of how a master node and a worker node interact w
 
 .. image:: ../images/manual/cluster/cluster_flow.png
```

Comment:
This image need an update. The File integrity thread is referenced as the Local integrity thread in the document.
![imagen](https://user-images.githubusercontent.com/47069802/148609824-4b5641ac-802f-484d-ae63-3b46a63bb88d.png)